### DISCUSSION TITLE
add MA version to interface

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [music-assistant/support#681] [Feature requests and ideas]
Now HACS has changed the resource info panels, and we need to click extra to go to the info panel on the resources, it is a bit of a pain to find the MA version through HACS.

why not add a meatball menu  to the menu bar top right, it's all empty space waiting for a purpose ;-)

<img width="1064" alt="Schermafbeelding 2022-07-16 om 13 37 37" src="https://user-images.githubusercontent.com/33354141/179353283-d36eb62e-ee13-477b-920e-56ed340c66a7.png">

and have the basic MA info available right in the Browser window.
It would certainly facilitate creating Issues in this Repo ;-)

HACS:

<img width="1063" alt="Schermafbeelding 2022-07-16 om 13 41 24" src="https://user-images.githubusercontent.com/33354141/179353377-c6600c5f-a7de-43f0-9f29-ea2c9446c923.png">

shows the meatballs to reveal other options, but posting here for reference

Another option could be to add it in the Integration panel:

<img width="342" alt="Schermafbeelding 2022-07-16 om 13 40 22" src="https://user-images.githubusercontent.com/33354141/179353337-10e83f6b-0842-444b-8078-06bcdb4fab3a.png">

but that would still require a lot more clicks....

**Comment by erkr** [comment]
I don't see versions in any of the Lovelace UI panels. But I like the idea to add it to the integration panel, or an about in the top right menu

**Comment by OzGav** [comment]
V1 question and the info is now available in the settings
